dot/network/host.go

```diff
@@ -262,32 +262,26 @@ func (h *host) bootstrap() {
 	}
 }
 
-// send writes the given message to the outbound message stream for the given
-// peer (gets the already opened outbound message stream or opens a new one).
-func (h *host) send(p peer.ID, pid protocol.ID, msg Message) (err error) {
-	// get outbound stream for given peer
-	s := h.getOutboundStream(p, pid)
```

Comment:
why is this removed?

Reply:
`send` is only called from `broadcastExcluding` when we don't have handshake data in which case we need to create a new stream.